### PR TITLE
Apply minor fixes to intro documentation

### PR DESCRIPTION
**Pull request by imrejonk**
This applies some minor style and natural language fixes to the intro pages in the Django documentation. I found them while following the intro tutorial.

This is my first commit to the Django project. I'm not sure if these kind of fixes are appreciated, feel free to close this PR if not.

Btw thanks for the great software and awesome documentation :)

### CODE REVIEW

**Review by smithdc1 — COMMENTED** [review]
Thanks for this. I think the last one is ok, but really not sure about the others. 

For example the quote marks are generally single ticks (e.g. the next line are single tick), so I think they should stay as they are.

File: docs/intro/tutorial07.txt [hunk 1]
```diff
@@ -299,7 +302,7 @@ Customizing your *project's* templates
 
 Create a ``templates`` directory in your project directory (the one that
 contains ``manage.py``). Templates can live anywhere on your filesystem that
-Django can access. (Django runs as whatever user your server runs.) However,
+Django can access (Django runs as whatever user your server runs as). However,
```
Comment (felixxm) [inline docs/intro/tutorial07.txt:305]: I think that's correct, the punctuation should go inside of the parentheses.

**Comment by felixxm** [comment]
Thanks for this patch, however I don't see much value in adding blank lines or changing to single quotes.

**Comment by imrejonk** [comment]
Understood, thanks for the feedback.

**Pull request closed without merging**